processing/src/main/java/io/druid/segment/filter/AndFilter.java

```diff
@@ -149,6 +150,17 @@ public boolean supportsBitmapIndex(BitmapIndexSelector selector)
   }
 
   @Override
+  public double estimateSelectivity(ColumnSelector columnSelector, BitmapIndexSelector indexSelector)
+  {
+    // Estimate selectivity with attribute value independence assumption
```

Comment:
Ah,I got your point. But this is a very basic assumption for estimating selectivity in databases and there are many more improvements (https://scholar.google.co.kr/scholar?newwindow=1&client=safari&hl=en-kr&um=1&ie=UTF-8&lr&q=related:A7vzlp4zVKU7uM:scholar.google.com/). I think we need to improve selectivity estimation later when druid is ready for providing more statistics like histogram, but it looks good for now. What do you think?